WHO.js

```diff
@@ -154,6 +149,36 @@ function scrape(doc, url) {
 	});
 }
 
+function detectSearch(item) {
+	return item.ISBN && ZU.cleanISBN(item.ISBN).startsWith('978924');
+}
+
+function doSearch(item) {
+	let ISBN = ZU.cleanISBN(item.ISBN);
+	let url = `https://apps.who.int/iris/discover/export?format=refman&list=discover&rpp=10&etal=0&query=${ISBN}&group_by=none&page=1`;
```

Comment:
Is the RIS quality comparable to the EM we get normally? If it's any lossier, it seems like we should just use the standard scraping routine.

Reply:
It's more or less equivalent. EM is missing books' editions, the RIS has `numPages` in the wrong field (but I'll fix that). I think RIS makes more sense for a search translator.